docs/contributing/commit-discipline.md

```diff
@@ -115,26 +115,36 @@ future contributors, and is no less important than the code you write. This
 section provides detailed guidance on how to write an excellent commit message.
 
 **Tip:** You can set up [Zulip's Git pre-commit hook][commit-hook] to
-automatically catch common mistakes in the commit message.
+automatically catch common commit message mistakes.
 
 [commit-hook]: ../git/zulip-tools.md#set-up-git-repo-script
 
 ### Commit summary, part 1
 
-The first part of the commit summary should only be 1-2 words, followed by a
-`:`. In most cases, it will describe the part of the product you are changing.
+The first part of the commit summary should only be 1-2 **lower-case**
+words, followed by a `:`, describing what the part of the product the
+commit changes. These prefixes are essential for maintainers to
+efficiently skim commits when doing release management or
+investigating regressions.
 
-Common examples include: settings, message feed, compose, left sidebar, right
-sidebar, recent (for **Recent conversations**), search, markdown, tooltips,
-popovers, drafts, integrations, email, docs, help, and api docs. In some cases,
-it's helpful to be a little more specific, e.g., emoji, spoilers, polls. If your
-commit doesn't cleanly map to a part of the product, you might use something
-like "css" for CSS-only changes, or the name of the file you refactored (not the
-full path).
+Common examples include: settings, message feed, compose, left
+sidebar, right sidebar, recent (for **Recent conversations**), search,
+markdown, tooltips, popovers, drafts, integrations, email, docs, help,
+and api docs; issue trackers can be a good source of inspiration (but
```

Comment:
What does this mean? Are you referring to labels in GitHub?

Reply:
(this bit about issue trackers should be either written more clearly or dropped)

Reply:
I think this was a reference to, if a PR fixes/addresses part of an issue in GitHub, then the issue title could be used as inspiration.

Reply:
dropping for now